src/app/bitstream-page/edit-bitstream-page/edit-bitstream-page.component.scss

```diff
@@ -6,3 +6,7 @@
     }
   }
 }
+:host ::ng-deep ds-dynamic-form-control-container > div > label {
```

Comment:
is it still necessary ?

Reply:
Yes. I checked this. It is still needed.